app/models/site.rb

```diff
@@ -24,4 +28,11 @@ def instance
       first_or_create
     end
   end
+
+  private
+
+    def update_banner_config
+      # If a site banner_image uploaded, then update Sufia's banner_image to use it
+      Sufia.config.banner_image = banner_image? ? banner_image.url : nil
```

Comment:
`Sufia.config` is how I'm updating Sufia's default configuration for its banner_image: https://github.com/projecthydra/sufia/blob/818cb4ca53d81fdb37835a0cd7c753d66eacd679/lib/sufia/configuration.rb#L10-L13

If there's a better way to achieve this, maybe I'm not aware of it (suggestions welcome!). Essentially, the goal here is to dynamically update Sufia's `banner_image` setting to the newly uploaded Lerna banner image location.